data/wacom-hid-5362.tablet

```diff
@@ -0,0 +1,17 @@
+# Wacom
+# Wacom HID 5362
+#
+# sysinfo.oMdWcZJ918.tar.gz
+# https://github.com/linuxwacom/wacom-hid-descriptors/issues/325
+
+[Device]
+Name=Wacom HID 5362 Pen,Wacom HID 5362 Finger
+ModelName=
+DeviceMatch=i2c:056a:5362
+IntegratedIn=Display;System
```

Comment:
Looks like `Width=` and `Height=` are missing here (that's the size it says on the box, in inches).

Reply:
I have added the dimensions according to the technical spec of the laptop.